connectivity/mbedtls/tests/UNITTESTS/doubles/mbedtls_stub.c

```diff
@@ -0,0 +1,434 @@
+/*
```

Comment:
is this a duplicate of mbed-os/connectivity/nanostack/coap-service/test/coap-service/unittest/stub/mbedtls_stub.c?

Reply:
I guess that is from the previous PR but same questions applies.

Reply:
I thought `connectivity/nanostack/coap-service` is from an external source and the coap-service unittest is not maintained as part of mbed os unit tests which is based on Makefile. mbedtls_stub.c and mbedtls_stub.h stub are used across lorawan, netsocket unit tests, so I made a copy of that as part of mbedtls stub creation and even which allow us to add/extent stub functionality if required. please correct if any of my understanding is wrong and share if you have any other suggestion

Reply:
connectivity/nanostack/coap-service/ is regularly synchronised with https://github.com/PelionIoT/coap-service. In my opinion we should not duplicate mbedtls_stub.c and mbedtls_stub.h, because if the repo is synced again, we may end up with new & old copies of the files. Personally I prefer making an exception in #14890 and add use the paths to the original coap-service stubs to connectivity/mbedtls/tests/UNITTESTS/doubles/CMakeLists.txt.

Reply:
@rajkan01 Thanks.
@paul-szczepanek-arm Please see the latest version of this PR, I think it solves our question.

Reply:
I have added using the paths to the original coap-service stubs to connectivity/mbedtls/tests/UNITTESTS/doubles/CMakeLists.txt.